prometheus-client/src/Prometheus/Label.hs

```diff
@@ -32,6 +32,9 @@ type Label0 = ()
 instance Label () where
     labelPairs () () = []
 
+instance Label [Text] where
```

Comment:
The downside to using a list is that you don't get compile time checking that the number of values match the number of labels.